libraries/quantity-picker-view/src/main/java/com/trendyol/uicomponents/quantitypickerview/QuantityPickerViewState.kt

```diff
@@ -158,6 +159,18 @@ data class QuantityPickerViewState(
         return copy(minQuantity = minQuantity, showLoading = false)
     }
 
+    internal fun getIconAddDrawable(icon: Drawable): QuantityPickerViewState {
```

Comment:
The other functions which create a new ViewState instance start with "getWith${fieldName}"
That's why I suggest we may continue to use the same pattern.

getIconAddDrawable -> getWithIconAddDrawable
getBackgroundDrawable -> getWithBackgroundDrawable
getAddButtonEnabled -> getWithAddButtonEnabled